Extensions/BitmapText/bitmaptextruntimeobject-pixi-renderer.js

```diff
@@ -59,7 +60,15 @@ gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.getRendererObject = function
   return this._pixiObject;
 };
 
-gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype._ensureFontAvailableAndGetFontName = function () {
+gdjs.BitmapTextRuntimeObjectPixiRenderer.prototype.onDestroy = function () {
+  //TODO: Mark the font from the object not used anymore
+
+  console.log('onDestroyFromScene');
+  console.log('Uninstall bitmapFont: ' + this._pixiObject._fontName);
+  this._bitmapFontManager.removeFontUsed(this._pixiObject._fontName);
```

Comment:
Because one object stopped using it doesn't mean other objects aren't too

Reply:
The name is maybe not perfectly chosen, but if you look at the implementation this is not unregistering the font as long as the reference count is not reaching 0 :) (this itself can be improved by keeping a cache of "dying fonts" and unload them only when there are too much of them in this cache, so that we don't always unload/reload a font if for some reason there is only a single object using the font being created/destroyed).